templates/taxi/car_list.html

```diff
@@ -1,13 +1,15 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list <a href="{% url "taxi:car-create"%}">+</a></h1>
   {% if car_list %}
     <ul>
       {% for car in car_list %}
         <li>
           <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
+          {{ car.model }} ({{ car.manufacturer.name }}) 
+          <a href="{% url "taxi:car-update" pk=car.id %} ">Update</a>
+          <a href="{% url "taxi:car-delete" pk=car.id %} ">Delete</a>
```

Comment:
There is an extra space at the end of the URL tag: `{% url "taxi:car-delete" pk=car.id %} `. It should be removed to ensure clean code.